attached_assets/Pasted--php-created-2025-02-01-03-57-23-var-www-html-public-legacy-config-php-sugar-config-a-1738728078455.txt
<?php
// created: 2025-02-01 03:57:23
// \var\www\html\public\legacy\config.php
$sugar_config = array (
  'addAjaxBannedModules' => 
  array (
    0 => 'SecurityGroups',
  ),
  'admin_access_control' => false,
  'admin_export_only' => false,
  'allowed_preview' => 
  array (
    0 => 'gif',
    1 => 'png',
    2 => 'jpeg',
    3 => 'jpg',
  ),
  'anti_malware_scanners' => 
  array (
    'SuiteCRM\\Utility\\AntiMalware\\Providers\\ClamTCP' => 
    array (
      'name' => 'ClamAntiVirus TCP',
      'support_page' => 'https://www.clamav.net/',
      'enabled' => false,
      'path' => NULL,
      'options' => 
      array (
        'ip' => '127.0.0.1',
        'port' => 3310,
        'type' => 'local',
      ),
    ),
    'SuiteCRM\\Utility\\AntiMalware\\Providers\\Sophos' => 
    array (
      'name' => 'Sophos Anti Virus (Linux)',
      'support_page' => 'https://www.sophos.com/en-us/products/free-tools/sophos-antivirus-for-linux.aspx',
      'enabled' => false,
      'path' => '/opt/sophos-av/bin/savscan',
      'options' => '-ss',
    ),
  ),
  'aop' => 
  array (
    'distribution_method' => 'roundRobin',
    'case_closure_email_template_id' => '2c3c4f5f-db09-6cf7-4635-679d00d74feb',
    'joomla_account_creation_email_template_id' => '35295037-b632-51e2-8109-679d0010658e',
    'case_creation_email_template_id' => '3df997d6-608a-f6a5-4a36-679d0087697f',
    'contact_email_template_id' => '4b347cf2-f61a-47ba-3e91-679d00fbc9a1',
    'user_email_template_id' => '549cff6b-e809-6373-92b1-679d007c3ded',
  ),
  'aos' => 
  array (
    'version' => '5.3.3',
    'contracts' => 
    array (
      'renewalReminderPeriod' => '14',
    ),
    'lineItems' => 
    array (
      'totalTax' => false,
      'enableGroups' => true,
    ),
    'invoices' => 
    array (
      'initialNumber' => '1',
    ),
    'quotes' => 
    array (
      'initialNumber' => '1',
    ),
  ),
  'cache_dir' => 'cache/',
  'calculate_response_time' => true,
  'calendar' => 
  array (
    'default_view' => 'week',
    'show_calls_by_default' => true,
    'show_tasks_by_default' => true,
    'show_completed_by_default' => true,
    'editview_width' => 990,
    'editview_height' => 485,
    'day_timestep' => 15,
    'week_timestep' => 30,
    'items_draggable' => true,
    'items_resizable' => true,
    'enable_repeat' => true,
    'max_repeat_count' => 1000,
  ),
  'chartEngine' => 'Jit',
  'common_ml_dir' => '',
  'create_default_user' => false,
  'cron' => 
  array (
    'max_cron_jobs' => 10,
    'max_cron_runtime' => 30,
    'min_cron_interval' => 30,
    'allowed_cron_users' => 
    array (
      0 => 'www-data',
    ),
  ),
  'dashlet_auto_refresh_min' => 30,
  'dashlet_display_row_options' => 
  array (
    0 => '1',
    1 => '3',
    2 => '5',
    3 => '10',
  ),
  'date_formats' => 
  array (
    'Y-m-d' => '2010-12-23',
    'm-d-Y' => '12-23-2010',
    'd-m-Y' => '23-12-2010',
    'Y/m/d' => '2010/12/23',
    'm/d/Y' => '12/23/2010',
    'd/m/Y' => '23/12/2010',
    'Y.m.d' => '2010.12.23',
    'd.m.Y' => '23.12.2010',
    'm.d.Y' => '12.23.2010',
  ),
  'datef' => 'm/d/Y',
  'dbUSRData' => 'same',
  'dbconfig' => 
  array (
    'db_host_name' => 'suitecrm_db',
    'db_host_instance' => '',
    'db_user_name' => 'suitecrm_user',
    'db_password' => 'Jamfinnarc1776!',
    'db_name' => 'suitecrm',
    'db_type' => 'mysql',
    'db_port' => '3306',
    'db_manager' => 'MysqliManager',
    'collation' => NULL,
    'charset' => NULL,
  ),
  'dbconfigoption' => 
  array (
    'persistent' => true,
    'autofree' => false,
    'debug' => 0,
    'ssl' => false,
  ),
  'default_action' => 'index',
  'default_charset' => NULL,
  'default_currency_iso4217' => 'USD',
  'default_currency_name' => 'US Dollar',
  'default_currency_significant_digits' => '2',
  'default_currency_symbol' => '$',
  'default_date_format' => 'Y-m-d',
  'default_decimal_seperator' => '.',
  'default_email_charset' => 'UTF-8',
  'default_email_client' => 'sugar',
  'default_email_editor' => 'html',
  'default_export_charset' => 'UTF-8',
  'default_language' => 'en_us',
  'default_locale_name_format' => 's f l',
  'default_max_tabs' => 8,
  'default_module' => 'Home',
  'default_module_access' => 
  array (
    'SecurityGroups' => false,
    'AOW_WorkFlow' => false,
  ),
  'default_module_favicon' => false,
  'default_navigation_paradigm' => 'm',
  'default_number_grouping_seperator' => ',',
  'default_password' => '',
  'default_permissions' => 
  array (
    'dir_mode' => 1528,
    'file_mode' => 493,
    'user' => '',
    'group' => '',
  ),
  'default_subpanel_links' => false,
  'default_subpanel_tabs' => true,
  'default_swap_last_viewed' => false,
  'default_swap_shortcuts' => false,
  'default_theme' => 'suite8',
  'default_time_format' => 'H:i',
  'default_user_is_admin' => false,
  'default_user_name' => '',
  'demoData' => 'no',
  'developerMode' => false,
  'disableAjaxUI' => true,
  'disable_convert_lead' => false,
  'disable_export' => false,
  'disable_persistent_connections' => false,
  'display_email_template_variable_chooser' => false,
  'display_inbound_email_buttons' => false,
  'dump_slow_queries' => false,
  'email_address_separator' => ',',
  'email_confirm_opt_in_email_template_id' => 'aef6515d-e038-900c-abbe-679d005a9aea',
  'email_default_client' => 'sugar',
  'email_default_delete_attachments' => true,
  'email_default_editor' => 'html',
  'email_enable_auto_send_opt_in' => false,
  'email_enable_confirm_opt_in' => 'not-opt-in',
  'email_warning_notifications' => true,
  'enable_action_menu' => true,
  'enable_line_editing_detail' => true,
  'enable_line_editing_list' => true,
  'enable_record_pagination' => true,
  'export_delimiter' => ',',
  'export_excel_compatible' => false,
  'filter_module_fields' => 
  array (
    'Users' => 
    array (
      0 => 'show_on_employees',
      1 => 'portal_only',
      2 => 'is_group',
      3 => 'system_generated_password',
      4 => 'external_auth_only',
      5 => 'sugar_login',
      6 => 'authenticate_id',
      7 => 'pwd_last_changed',
      8 => 'is_admin',
      9 => 'user_name',
      10 => 'user_hash',
      11 => 'password',
      12 => 'last_login',
      13 => 'oauth_tokens',
    ),
    'Employees' => 
    array (
      0 => 'show_on_employees',
      1 => 'portal_only',
      2 => 'is_group',
      3 => 'system_generated_password',
      4 => 'external_auth_only',
      5 => 'sugar_login',
      6 => 'authenticate_id',
      7 => 'pwd_last_changed',
      8 => 'is_admin',
      9 => 'user_name',
      10 => 'user_hash',
      11 => 'password',
      12 => 'last_login',
      13 => 'oauth_tokens',
    ),
  ),
  'google_auth_json' => '',
  'hide_subpanels' => true,
  'history_max_viewed' => 50,
  'host_name' => '4.236.188.48',
  'id_validation_pattern' => '/^[a-zA-Z0-9_-]*$/i',
  'imap_test' => false,
  'import_max_execution_time' => 3600,
  'import_max_records_per_file' => 100,
  'import_max_records_total_limit' => '',
  'installer_locked' => true,
  'jobs' => 
  array (
    'min_retry_interval' => 30,
    'max_retries' => 5,
    'timeout' => 86400,
  ),
  'js_custom_version' => 1,
  'js_lang_version' => 1,
  'languages' => 
  array (
    'en_us' => 'English (US)',
  ),
  'large_scale_test' => false,
  'lead_conv_activity_opt' => 'donothing',
  'legacy_email_behaviour' => false,
  'list_max_entries_per_modal' => 10,
  'list_max_entries_per_page' => 20,
  'list_max_entries_per_subpanel' => 10,
  'listview_max_height' => 0,
  'listview_pagination_type' => 'pagination',
  'lock_default_user_name' => false,
  'lock_homepage' => false,
  'lock_subpanels' => false,
  'log_dir' => '.',
  'log_file' => 'suitecrm.log',
  'log_memory_usage' => false,
  'logger' => 
  array (
    'level' => 'fatal',
    'file' => 
    array (
      'ext' => '.log',
      'name' => 'suitecrm',
      'dateFormat' => '%c',
      'maxSize' => '10MB',
      'maxLogs' => 10,
      'suffix' => '',
    ),
  ),
  'login_language' => true,
  'max_dashlets_homepage' => '15',
  'name_formats' => 
  array (
    's f l' => 's f l',
    'f l' => 'f l',
    's l' => 's l',
    'l, s f' => 'l, s f',
    'l, f' => 'l, f',
    's l, f' => 's l, f',
    'l s f' => 'l s f',
    'l f s' => 'l f s',
  ),
  'oauth2_encryption_key' => 'gwIbPri9ERSeElLD8SmHzjHm2+pQvZxqY2kyrJ8Tuj0=',
  'passwordsetting' => 
  array (
    'SystemGeneratedPasswordON' => '',
    'generatepasswordtmpl' => 'cb154bac-9699-7831-0b81-679d004b4ea0',
    'lostpasswordtmpl' => 'd411b5e6-40cd-a3f9-7879-679d00eef734',
    'factoremailtmpl' => 'ddeda2cc-f0be-f8fb-bbf6-679d00b96f9d',
    'forgotpasswordON' => false,
    'linkexpiration' => '1',
    'linkexpirationtime' => '30',
    'linkexpirationtype' => '1',
    'systexpiration' => '1',
    'systexpirationtime' => '7',
    'systexpirationtype' => '1',
    'systexpirationlogin' => '',
  ),
  'pdf' => 
  array (
    'defaultEngine' => 'TCPDFEngine',
  ),
  'portal_view' => 'single_user',
  'record_modal_pagination_type' => 'pagination',
  'require_accounts' => true,
  'resource_management' => 
  array (
    'special_query_limit' => 50000,
    'special_query_modules' => 
    array (
      0 => 'AOR_Reports',
      1 => 'Export',
      2 => 'Import',
      3 => 'Administration',
      4 => 'Sync',
    ),
    'default_limit' => 20000,
  ),
  'rss_cache_time' => '10800',
  'save_query' => 'all',
  'search' => 
  array (
    'controller' => 'UnifiedSearch',
    'defaultEngine' => 'BasicSearchEngine',
    'pagination' => 
    array (
      'min' => 10,
      'max' => 50,
      'step' => 10,
    ),
    'ElasticSearch' => 
    array (
      'enabled' => false,
      'host' => 'localhost',
      'user' => '',
      'pass' => '',
      'index' => '1a5362a11e56b4ad243b9f58425cced0',
      'search_wildcard_char' => '%',
      'search_wildcard_infront' => true,
    ),
  ),
  'search_wildcard_char' => '%',
  'search_wildcard_infront' => false,
  'securitysuite_additive' => true,
  'securitysuite_filter_user_list' => false,
  'securitysuite_inherit_assigned' => true,
  'securitysuite_inherit_creator' => true,
  'securitysuite_inherit_parent' => true,
  'securitysuite_popup_select' => false,
  'securitysuite_strict_rights' => false,
  'securitysuite_user_popup' => true,
  'securitysuite_user_role_precedence' => true,
  'securitysuite_version' => '6.5.17',
  'session_dir' => '',
  'session_gc' => 
  array (
    'enable' => true,
    'gc_probability' => 1,
    'gc_divisor' => 100,
  ),
  'showDetailData' => true,
  'showThemePicker' => true,
  'site_url' => 'http://4.236.188.48',
  'slow_query_time_msec' => '100',
  'snooze_alert_timer' => 600,
  'stackTrace' => false,
  'stack_trace_errors' => false,
  'strict_id_validation' => false,
  'subpanel_max_height' => 620,
  'subpanel_pagination_type' => 'pagination',
  'sugar_version' => '6.5.25',
  'sugarbeet' => false,
  'suitecrm_version' => '8.7.1',
  'system_email_templates' => 
  array (
    'confirm_opt_in_template_id' => 'aef6515d-e038-900c-abbe-679d005a9aea',
  ),
  'system_name' => 'SuiteCRM',
  'time_formats' => 
  array (
    'H:i' => '23:00',
    'h:ia' => '11:00pm',
    'h:iA' => '11:00PM',
    'h:i a' => '11:00 pm',
    'h:i A' => '11:00 PM',
    'H.i' => '23.00',
    'h.ia' => '11.00pm',
    'h.iA' => '11.00PM',
    'h.i a' => '11.00 pm',
    'h.i A' => '11.00 PM',
  ),
  'timef' => 'H:i',
  'tmp_dir' => 'cache/xml/',
  'tracker_max_display_length' => 15,
  'translation_string_prefix' => false,
  'trusted_hosts' => 
  array (
  ),
  'unique_key' => '1a5362a11e56b4ad243b9f58425cced0',
  'upload_badext' => 
  array (
    0 => 'php',
    1 => 'php3',
    2 => 'php4',
    3 => 'php5',
    4 => 'php6',
    5 => 'php7',
    6 => 'php8',
    7 => 'pl',
    8 => 'cgi',
    9 => 'py',
    10 => 'asp',
    11 => 'cfm',
    12 => 'js',
    13 => 'vbs',
    14 => 'html',
    15 => 'htm',
    16 => 'phtml',
    17 => 'phar',
  ),
  'upload_dir' => 'upload/',
  'upload_maxsize' => 30000000,
  'use_common_ml_dir' => false,
  'use_real_names' => true,
  'valid_image_ext' => 
  array (
    0 => 'gif',
    1 => 'png',
    2 => 'jpg',
    3 => 'jpeg',
    4 => 'svg',
    5 => 'bmp',
  ),
  'valid_imap_ports' => 
  array (
    0 => '110',
    1 => '143',
    2 => '993',
    3 => '995',
  ),
  'vcal_time' => '2',
  'verify_client_ip' => true,
  'web_to_lead_allowed_redirect_hosts' => 
  array (
  ),
);

attached_assets/config.php
<?php
// created: 2025-02-16 14:01:08
$Diagnostic = array (
  'addAjaxBannedModules' => 
  array (
    0 => 'SecurityGroups',
  ),
  'admin_access_control' => false,
  'admin_export_only' => false,
  'allowed_preview' => 
  array (
    0 => 'gif',
    1 => 'png',
    2 => 'jpeg',
    3 => 'jpg',
  ),
  'anti_malware_scanners' => 
  array (
    'SuiteCRM\\Utility\\AntiMalware\\Providers\\ClamTCP' => 
    array (
      'name' => 'ClamAntiVirus TCP',
      'support_page' => 'https://www.clamav.net/',
      'enabled' => false,
      'path' => NULL,
      'options' => 
      array (
        'ip' => '127.0.0.1',
        'port' => 3310,
        'type' => 'local',
      ),
    ),
    'SuiteCRM\\Utility\\AntiMalware\\Providers\\Sophos' => 
    array (
      'name' => 'Sophos Anti Virus (Linux)',
      'support_page' => 'https://www.sophos.com/en-us/products/free-tools/sophos-antivirus-for-linux.aspx',
      'enabled' => false,
      'path' => '/opt/sophos-av/bin/savscan',
      'options' => '-ss',
    ),
  ),
  'aop' => 
  array (
    'distribution_method' => 'roundRobin',
    'case_closure_email_template_id' => '7f7e8c38-8c75-18de-be98-67ae269cad8b',
    'joomla_account_creation_email_template_id' => '8465bec1-9d39-a045-69ab-67ae261c2179',
    'case_creation_email_template_id' => '88e94922-ba09-d0f9-94c3-67ae26fe6351',
    'contact_email_template_id' => '8ebd247f-59da-4e21-d40f-67ae262a77a9',
    'user_email_template_id' => '99096a2e-0a2f-17aa-04c2-67ae26287423',
  ),
  'aos' => 
  array (
    'version' => '5.3.3',
    'contracts' => 
    array (
      'renewalReminderPeriod' => '14',
    ),
    'lineItems' => 
    array (
      'totalTax' => false,
      'enableGroups' => true,
    ),
    'invoices' => 
    array (
      'initialNumber' => '1',
    ),
    'quotes' => 
    array (
      'initialNumber' => '1',
    ),
  ),
  'cache_dir' => 'cache/',
  'calculate_response_time' => true,
  'calendar' => 
  array (
    'default_view' => 'week',
    'show_calls_by_default' => true,
    'show_tasks_by_default' => true,
    'show_completed_by_default' => true,
    'editview_width' => 990,
    'editview_height' => 485,
    'day_timestep' => 15,
    'week_timestep' => 30,
    'items_draggable' => true,
    'items_resizable' => true,
    'enable_repeat' => true,
    'max_repeat_count' => 1000,
  ),
  'chartEngine' => 'Jit',
  'common_ml_dir' => '',
  'create_default_user' => false,
  'cron' => 
  array (
    'max_cron_jobs' => 10,
    'max_cron_runtime' => 30,
    'min_cron_interval' => 30,
  ),
  'dashlet_auto_refresh_min' => 30,
  'dashlet_display_row_options' => 
  array (
    0 => '1',
    1 => '3',
    2 => '5',
    3 => '10',
  ),
  'date_formats' => 
  array (
    'Y-m-d' => '2010-12-23',
    'm-d-Y' => '12-23-2010',
    'd-m-Y' => '23-12-2010',
    'Y/m/d' => '2010/12/23',
    'm/d/Y' => '12/23/2010',
    'd/m/Y' => '23/12/2010',
    'Y.m.d' => '2010.12.23',
    'd.m.Y' => '23.12.2010',
    'm.d.Y' => '12.23.2010',
  ),
  'datef' => 'm/d/Y',
  'dbconfig' => 
  array (
    'db_host_name' => 'mariadb',
    'db_host_instance' => '',
    'db_user_name' => 'bn_suitecrm',
    'db_password' => '********',
    'db_name' => 'bitnami_suitecrm',
    'db_type' => 'mysql',
    'db_port' => '3306',
    'db_manager' => 'MysqliManager',
    'collation' => NULL,
    'charset' => NULL,
  ),
  'dbconfigoption' => 
  array (
    'persistent' => true,
    'autofree' => false,
    'debug' => 0,
    'ssl' => false,
  ),
  'default_action' => 'index',
  'default_charset' => NULL,
  'default_currency_iso4217' => 'USD',
  'default_currency_name' => 'US Dollar',
  'default_currency_significant_digits' => '2',
  'default_currency_symbol' => '$',
  'default_date_format' => 'Y-m-d',
  'default_decimal_seperator' => '.',
  'default_email_charset' => 'UTF-8',
  'default_email_client' => 'sugar',
  'default_email_editor' => 'html',
  'default_export_charset' => 'ISO-8859-1',
  'default_language' => 'en_us',
  'default_locale_name_format' => 's f l',
  'default_max_tabs' => 8,
  'default_module' => 'Home',
  'default_module_access' => 
  array (
    'SecurityGroups' => false,
    'AOW_WorkFlow' => false,
  ),
  'default_module_favicon' => false,
  'default_navigation_paradigm' => 'm',
  'default_number_grouping_seperator' => ',',
  'default_password' => '',
  'default_permissions' => 
  array (
    'dir_mode' => 1528,
    'file_mode' => 493,
    'user' => '',
    'group' => '',
  ),
  'default_subpanel_links' => false,
  'default_subpanel_tabs' => true,
  'default_swap_last_viewed' => false,
  'default_swap_shortcuts' => false,
  'default_theme' => 'suite8',
  'default_time_format' => 'H:i',
  'default_user_is_admin' => false,
  'default_user_name' => '',
  'demoData' => 'no',
  'developerMode' => false,
  'disableAjaxUI' => true,
  'disable_convert_lead' => false,
  'disable_export' => false,
  'disable_persistent_connections' => false,
  'display_email_template_variable_chooser' => false,
  'display_inbound_email_buttons' => false,
  'dump_slow_queries' => false,
  'email_address_separator' => ',',
  'email_confirm_opt_in_email_template_id' => '33925382-517b-0854-d84b-67ae262c07d3',
  'email_default_client' => 'sugar',
  'email_default_delete_attachments' => true,
  'email_default_editor' => 'html',
  'email_enable_auto_send_opt_in' => false,
  'email_enable_confirm_opt_in' => 'not-opt-in',
  'email_warning_notifications' => true,
  'enable_action_menu' => true,
  'enable_line_editing_detail' => true,
  'enable_line_editing_list' => true,
  'enable_record_pagination' => true,
  'export_delimiter' => ',',
  'export_excel_compatible' => false,
  'filter_module_fields' => 
  array (
    'Users' => 
    array (
      0 => 'show_on_employees',
      1 => 'portal_only',
      2 => 'is_group',
      3 => 'system_generated_password',
      4 => 'external_auth_only',
      5 => 'sugar_login',
      6 => 'authenticate_id',
      7 => 'pwd_last_changed',
      8 => 'is_admin',
      9 => 'user_name',
      10 => 'user_hash',
      11 => 'password',
      12 => 'last_login',
      13 => 'oauth_tokens',
    ),
    'Employees' => 
    array (
      0 => 'show_on_employees',
      1 => 'portal_only',
      2 => 'is_group',
      3 => 'system_generated_password',
      4 => 'external_auth_only',
      5 => 'sugar_login',
      6 => 'authenticate_id',
      7 => 'pwd_last_changed',
      8 => 'is_admin',
      9 => 'user_name',
      10 => 'user_hash',
      11 => 'password',
      12 => 'last_login',
      13 => 'oauth_tokens',
    ),
  ),
  'google_auth_json' => '',
  'hide_subpanels' => true,
  'history_max_viewed' => 50,
  'host_name' => 'localhost',
  'id_validation_pattern' => '/^[a-zA-Z0-9_-]*$/i',
  'imap_test' => false,
  'import_max_execution_time' => 3600,
  'import_max_records_per_file' => 100,
  'import_max_records_total_limit' => '',
  'installed' => true,
  'installer_locked' => true,
  'jobs' => 
  array (
    'min_retry_interval' => 30,
    'max_retries' => 5,
    'timeout' => 86400,
  ),
  'js_custom_version' => 1,
  'js_lang_version' => 1,
  'languages' => 
  array (
    'en_us' => 'English (US)',
  ),
  'large_scale_test' => false,
  'lead_conv_activity_opt' => 'donothing',
  'legacy_email_behaviour' => false,
  'list_max_entries_per_modal' => 10,
  'list_max_entries_per_page' => 20,
  'list_max_entries_per_subpanel' => 10,
  'listview_max_height' => 0,
  'listview_pagination_type' => 'pagination',
  'lock_default_user_name' => false,
  'lock_homepage' => false,
  'lock_subpanels' => false,
  'log_dir' => '../../logs/legacy',
  'log_file' => 'suitecrm.log',
  'log_memory_usage' => false,
  'logger' => 
  array (
    'level' => 'fatal',
    'file' => 
    array (
      'ext' => '.log',
      'name' => 'suitecrm',
      'dateFormat' => '%c',
      'maxSize' => '10MB',
      'maxLogs' => 10,
      'suffix' => '',
    ),
  ),
  'login_language' => true,
  'max_dashlets_homepage' => '15',
  'name_formats' => 
  array (
    's f l' => 's f l',
    'f l' => 'f l',
    's l' => 's l',
    'l, s f' => 'l, s f',
    'l, f' => 'l, f',
    's l, f' => 's l, f',
    'l s f' => 'l s f',
    'l f s' => 'l f s',
  ),
  'oauth2_encryption_key' => 'IF2SZD8M92/u2DG1t8q+YT+kVfmuS2eJ1RkAcYkdm8E=',
  'passwordsetting' => 
  array (
    'SystemGeneratedPasswordON' => '',
    'generatepasswordtmpl' => '4cd8b5ab-ef87-7ca5-93ef-67ae261a333e',
    'lostpasswordtmpl' => '552821ec-22a5-bf48-51e4-67ae264e174f',
    'factoremailtmpl' => '5bcdf0f0-e547-6eb3-077e-67ae26ccf4e5',
    'forgotpasswordON' => false,
    'linkexpiration' => '1',
    'linkexpirationtime' => '30',
    'linkexpirationtype' => '1',
    'systexpiration' => '1',
    'systexpirationtime' => '7',
    'systexpirationtype' => '1',
    'systexpirationlogin' => '',
  ),
  'pdf' => 
  array (
    'defaultEngine' => 'TCPDFEngine',
  ),
  'portal_view' => 'single_user',
  'record_modal_pagination_type' => 'pagination',
  'require_accounts' => true,
  'resource_management' => 
  array (
    'special_query_limit' => 50000,
    'special_query_modules' => 
    array (
      0 => 'AOR_Reports',
      1 => 'Export',
      2 => 'Import',
      3 => 'Administration',
      4 => 'Sync',
    ),
    'default_limit' => 20000,
  ),
  'rss_cache_time' => '10800',
  'save_query' => 'all',
  'search' => 
  array (
    'controller' => 'UnifiedSearch',
    'defaultEngine' => 'BasicSearchEngine',
    'pagination' => 
    array (
      'min' => 10,
      'max' => 50,
      'step' => 10,
    ),
    'ElasticSearch' => 
    array (
      'enabled' => false,
      'host' => 'localhost',
      'user' => '',
      'pass' => '',
      'index' => '65486d89e52b61195db14bb4e808bdbe',
      'search_wildcard_char' => '%',
      'search_wildcard_infront' => true,
    ),
  ),
  'search_wildcard_char' => '%',
  'search_wildcard_infront' => false,
  'securitysuite_additive' => true,
  'securitysuite_filter_user_list' => false,
  'securitysuite_inherit_assigned' => true,
  'securitysuite_inherit_creator' => true,
  'securitysuite_inherit_parent' => true,
  'securitysuite_popup_select' => false,
  'securitysuite_strict_rights' => false,
  'securitysuite_user_popup' => true,
  'securitysuite_user_role_precedence' => true,
  'securitysuite_version' => '6.5.17',
  'session_dir' => '',
  'session_gc' => 
  array (
    'enable' => true,
    'gc_probability' => 1,
    'gc_divisor' => 100,
  ),
  'showDetailData' => true,
  'showThemePicker' => true,
  'site_url' => 'http://localhost:8080',
  'slow_query_time_msec' => '100',
  'snooze_alert_timer' => 600,
  'stackTrace' => false,
  'stack_trace_errors' => false,
  'strict_id_validation' => false,
  'subpanel_max_height' => 620,
  'subpanel_pagination_type' => 'pagination',
  'sugar_version' => NULL,
  'sugarbeet' => false,
  'suitecrm_version' => NULL,
  'system_email_templates' => 
  array (
    'confirm_opt_in_template_id' => '33925382-517b-0854-d84b-67ae262c07d3',
  ),
  'system_name' => 'SuiteCRM',
  'time_formats' => 
  array (
    'H:i' => '23:00',
    'h:ia' => '11:00pm',
    'h:iA' => '11:00PM',
    'h:i a' => '11:00 pm',
    'h:i A' => '11:00 PM',
    'H.i' => '23.00',
    'h.ia' => '11.00pm',
    'h.iA' => '11.00PM',
    'h.i a' => '11.00 pm',
    'h.i A' => '11.00 PM',
  ),
  'timef' => 'H:i',
  'tmp_dir' => 'cache/xml/',
  'tracker_max_display_length' => 15,
  'translation_string_prefix' => false,
  'trusted_hosts' => 
  array (
  ),
  'unique_key' => '65486d89e52b61195db14bb4e808bdbe',
  'upload_badext' => 
  array (
    0 => 'php',
    1 => 'php3',
    2 => 'php4',
    3 => 'php5',
    4 => 'php6',
    5 => 'php7',
    6 => 'php8',
    7 => 'pl',
    8 => 'cgi',
    9 => 'py',
    10 => 'asp',
    11 => 'cfm',
    12 => 'js',
    13 => 'vbs',
    14 => 'html',
    15 => 'htm',
    16 => 'phtml',
    17 => 'phar',
  ),
  'upload_dir' => 'upload/',
  'upload_maxsize' => 30000000,
  'use_common_ml_dir' => false,
  'use_real_names' => true,
  'valid_image_ext' => 
  array (
    0 => 'gif',
    1 => 'png',
    2 => 'jpg',
    3 => 'jpeg',
    4 => 'svg',
    5 => 'bmp',
  ),
  'valid_imap_ports' => 
  array (
    0 => '110',
    1 => '143',
    2 => '993',
    3 => '995',
  ),
  'vcal_time' => '2',
  'verify_client_ip' => true,
  'web_to_lead_allowed_redirect_hosts' => 
  array (
  ),
  'email_allow_send_as_user' => false,
  'email_xss' => 'YToxMzp7czo2OiJhcHBsZXQiO3M6NjoiYXBwbGV0IjtzOjQ6ImJhc2UiO3M6NDoiYmFzZSI7czo1OiJlbWJlZCI7czo1OiJlbWJlZCI7czo0OiJmb3JtIjtzOjQ6ImZvcm0iO3M6NToiZnJhbWUiO3M6NToiZnJhbWUiO3M6ODoiZnJhbWVzZXQiO3M6ODoiZnJhbWVzZXQiO3M6NjoiaWZyYW1lIjtzOjY6ImlmcmFtZSI7czo2OiJpbXBvcnQiO3M6ODoiXD9pbXBvcnQiO3M6NToibGF5ZXIiO3M6NToibGF5ZXIiO3M6NDoibGluayI7czo0OiJsaW5rIjtzOjY6Im9iamVjdCI7czo2OiJvYmplY3QiO3M6MzoieG1wIjtzOjM6InhtcCI7czo2OiJzY3JpcHQiO3M6Njoic2NyaXB0Ijt9',
);